content/docs/5.1/deploy/install/production.md

```diff
@@ -148,7 +148,7 @@ For example, execute the following command to check the status of the `tikv-test
 tiup cluster display tikv-test
 ```
 
-Expected output includes the instance ID, role, host, listening port, and status (because the cluster is not started yet, so the status is `Down`/`inactive`), and directory information.
+The command output should include the instance ID, role, host, listening port, and status (the status of the cluster is `Down`/`inactive` because it is not started yet), and directory information.
```

Comment:
@shichun-0415 Maybe we can remove "should".

Reply:
Agree